tests/data/conversion_test.py

```diff
@@ -6,13 +6,12 @@
 class TestConversion(test_util.ConverterTestSuite):
     def test_determine_area_factors(self):
         assertions = [
-            (['0', '0'], (1.0, 1.0)),
-            (['0', '1'], (1.0, 0.1)),
-            (['1', '0'], (0.1, 1.0)),
-            (['2', '4'], (0.2, 0.4)),
```

Comment:
Restore this test.

Reply:
Only line 12 or the whole test?

Reply:
Only line 12, others are included already.

Reply:
done